crates/ra_lsp_server/src/caps.rs

```diff
@@ -57,6 +59,49 @@ pub fn server_capabilities() -> ServerCapabilities {
         execute_command_provider: None,
         workspace: None,
         call_hierarchy_provider: Some(CallHierarchyServerCapability::Simple(true)),
+        semantic_tokens_provider: Some(SemanticTokensServerCapabilities::SemanticTokensOptions(
```

Comment:
I'm thinking the token and modifier lists should be exposed in a `semantic_tokens.rs` file in `ra_ide` so they can be reused in `conv.rs`. Maybe as arrays or strings that then get converted to `SemanticTokenType` and `SemanticTokenModifer` via `into()` when we fill in the server caps?